nemo/export/quantize/quantizer.py

```diff
@@ -197,7 +202,7 @@ def forward_loop():
                     print(f"Calibrating batch {i}")
                 model.predict_step(batch, i)
 
-        model = atq.quantize(model, self.atq_config, forward_loop)
+        model = mtq.quantize(model, self.quant_cfg, forward_loop)
```

Comment:
Please note that `def forward_loop(model): ...` is recommended over `def forward_loop(): ...` and the latter one will be deprecated soon

Reply:
changed